packages/sanes-chrome-extension/src/routes/share-identity/test/operateShareIdentity.ts

```diff
@@ -0,0 +1,81 @@
+import TestUtils from 'react-dom/test-utils';
+import { findRenderedDOMComponentWithId } from '../../../utils/test/reactElemFinder';
+import { SHARE_IDENTITY_ACCEPT } from '../components/AcceptRequest';
+import { SHARE_IDENTITY_SHOW } from '../components/ShowRequest';
+import { SHARE_IDENTITY_REJECT } from '../components/RejectRequest';
+
+export const clickOnAcceptButton = async (ShareIdentityDom: React.Component): Promise<void> => {
+  const inputs = TestUtils.scryRenderedDOMComponentsWithTag(ShareIdentityDom, 'button');
+
+  expect(inputs.length).toBe(2);
+
+  const acceptButton = inputs[0];
+
+  TestUtils.act(
+    (): void => {
```

Comment:
We changed the linter settings, such that the return type only needs to be set when the function has a name. Especially in test code, you don't need to set it. This saves you 2 lines and one indent level:

```ts
TestUtils.act(() => {
  TestUtils.Simulate.click(acceptButton);
});
```